### PR TITLE
[Feedbacksystem] Improve documentation

### LINKED ISSUE #22: [Feedbacksystem] Improve documentation
The documentation of the Feedbasystem Helm Chart should be improved as it does not contain much information and some parts are out of date.

### PR DESCRIPTION
**Pull request by mxsph**
resolves #22

### CODE REVIEW

File: charts/feedbacksystem/README.md [hunk 1]
```diff
@@ -1,36 +1,120 @@
...
+
+#### Mysql
+
+| Parameter                       | Description                            | Default   |
+| ------------------------------- | -------------------------------------- | --------- |
+| mysql.auth.database             | The name of the Database               | fbs       |
+| mysql.auth.username             | The name of the Database User          | fbs       |
+| mysql.auth.password             | The Password of the Database User      | fbs       |
+| mysql.auth.rootPassword         | The Password of the Database Root User | root      |
+| mysql.primray.existingConfigmap | **TODO**                               | CHANGE-ME |
```
Comment (mxsph) [inline charts/feedbacksystem/README.md:73]: @Zitrone44 I didn't know exactly what this configuration was for, can you add a description here?

File: charts/feedbacksystem/README.md [hunk 2]
```diff
@@ -1,36 +1,120 @@
...
+
+#### Runner Mysql
+
+| Parameter                             | Description                            | Default   |
+| ------------------------------------- | -------------------------------------- | --------- |
+| runnerMysql.auth.database             | The name of the Database               | fbs       |
+| runnerMysql.auth.username             | The name of the Database User          | fbs       |
+| runnerMysql.auth.password             | The Password of the Database User      | fbs       |
+| runnerMysql.auth.rootPassword         | The Password of the Database Root User | root      |
+| runnerMysql.primray.existingConfigmap | **TODO**                               | CHANGE-ME |
```
Comment (mxsph) [inline charts/feedbacksystem/README.md:83]: @Zitrone44 I didn't know exactly what this configuration was for, can you add a description here?

File: charts/feedbacksystem/README.md (hunk 1 above)
Comment (Zitrone44) [inline charts/feedbacksystem/README.md:73]:
```suggestion
| Parameter                       | Description                                         | Default   |
| ------------------------------- |:--------------------------------------------------- | --------- |
| mysql.auth.database             | The name of the Database                            | fbs       |
| mysql.auth.username             | The name of the Database User                       | fbs       |
| mysql.auth.password             | The Password of the Database User                   | fbs       |
| mysql.auth.rootPassword         | The Password of the Database Root User              | root      |
| mysql.primray.existingConfigmap | The name of an existing ConfigMap containg `my.cnf` | CHANGE-ME |
```

File: charts/feedbacksystem/README.md (hunk 2 above)
Comment (Zitrone44) [inline charts/feedbacksystem/README.md:83]:
```suggestion
| Parameter                             | Description                                         | Default   |
| ------------------------------------- | --------------------------------------------------- | --------- |
| runnerMysql.auth.database             | The name of the Database                            | fbs       |
| runnerMysql.auth.username             | The name of the Database User                       | fbs       |
| runnerMysql.auth.password             | The Password of the Database User                   | fbs       |
| runnerMysql.auth.rootPassword         | The Password of the Database Root User              | root      |
| runnerMysql.primray.existingConfigmap | The name of an existing ConfigMap containg `my.cnf` | CHANGE-ME |
```

**Review by Zitrone44 — APPROVED** [review]

**Pull request merged**